roads.mss

```diff
@@ -393,6 +394,25 @@ as well. */
 }
 
 /* ================================================================== */
+/* FERRIES
+/* ================================================================== */
+
+#ferry {
```

Comment:
@yohanboniface now that you mention it, I do have it filtered out in the SQL! I can simplify this quite a bit, then